kv/kv.go

```diff
@@ -85,6 +85,8 @@ type Retriever interface {
 	// The Iterator must be Closed after use.
 	Seek(k Key) (Iterator, error)
 
+	SeekWithBatchSize(k Key, batchSize int) (Iterator, error)
```

Comment:
You can add a interface 
```
type Seeker interface{
    SeekWithBatchSize(k Key, batchSize int) (Iterator, error)
}```, then just implement it in tikvSnapshot is ok.

Reply:
wonderful idea~